nunit.tests.vb/TestClass.vb

```diff
@@ -1,12 +1,12 @@
 ﻿Option Explicit On
-
+' NUnit3 Tests,  see documentation : https://github.com/nunit/docs/wiki/NUnit-Documentation 
```

Comment:
Trailing whitespace, double space after comma, and space before colon, and I wouldn't capitalize `Tests`. I'd prefer to make two sentences instead of the comma.